src/Core/Managed/Shared/Utils.cs

```diff
@@ -74,6 +74,20 @@ public static TimeSpan ValidateDuration(string value)
             return interval;
         }
 
+        /// <summary>
+        /// Returns min DateTimeOffset value if not a valid DateTimeOffset.
+        /// </summary>
+        public static DateTimeOffset ValidateDateTimeOffset(string value)
+        {
+            DateTimeOffset timestamp;
+            if (!DateTimeOffset.TryParse(value, CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal, out timestamp))
```

Comment:
Since it is a server side SDK I would assume that everything needs to be in UTC. So I would change that in RequestTelemetry as well.

---
In reply to: [72862498](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/266#discussion_r72862498) [](ancestors = 72862498)